packages/snap/src/polkadot/events/index.ts

```diff
@@ -1,19 +1,29 @@
-import {Callback, HexHash, Origin, PolkadotEventArgument, TxEventArgument} from "@nodefactory/metamask-polkadot-types";
-import {EventEmitterImplementation} from "./emitter";
-
-export interface EventEmitter<K, T, J>  {
-  addListener(event: K, identifier: T, listener: Callback<J>): this;
-  removeListener(event: K, identifier: T, listener: Callback<J>): this;
-  removeAllListeners(event: K, identifier: T): this;
-  getListenersCount(event: K, identifier: T): number;
-  emit(event: K, identifier: T, arg: J): boolean;
+import {TxStatus} from "@nodefactory/metamask-polkadot-types";
+import {EventEmitter} from "events";
+import {StrictEmitterWithOriginProvider} from "./emitter";
+import StrictEventEmitter from "strict-event-emitter-types";
+
+// Polkadot event emitters
+
+interface PolkadotEvents {
+  onBalanceChange: string;
```

Comment:
isn't this suppose to be
(balance: string) => void

It would maybe make sense for name of event to just be "balance" since it's used .on("balance", ...) ?